src/components/search/_macro-options.md

```diff
@@ -1,4 +1,11 @@
-| Name         | Type                                   | Required | Description              |
-| ------------ | -------------------------------------- | -------- | ------------------------ |
-| input        | `Input` [_(ref)_](/components/input)   | true     | Settings for the input.  |
-| searchButton | `Button` [_(ref)_](/components/button) | true     | Settings for the button. |
+| Name         | Type                                 | Required | Description                                    |
+| ------------ | ------------------------------------ | -------- | ---------------------------------------------- |
+| input        | `Input` [_(ref)_](/components/input) | true     | Settings for the input.                        |
+| searchButton | object`<searchButton>`               | true     | Settings for the [searchButton](#searchButton) |
+
+## searchButton
+
+| Name                   | Type                                   | Required | Description                                                   |
+| ---------------------- | -------------------------------------- | -------- | ------------------------------------------------------------- |
+| Button configuration   | `Button` [_(ref)_](/components/button) | true     | Settings for the button.                                      |
```

Comment:
Unless we're importing the whole component with all params, we usually list out here which of the options you can set here, so it's clear what can be changed. Otherwise a user might try to set a param that hasn't been used here, and report the component is broken. So in this case: type, id, classes, attributes, and iconType.